### ISSUE TITLE
Generator produces empty debug message on tutorial Async API file

### ISSUE THREAD

**Issue by KatzeMitHut** [event-catalog/generator-asyncapi#26]
I tried the generator with the AsyncAPI file from this Avro tutorial https://www.asyncapi.com/docs/tutorials/kafka/configure-kafka-avro.
I used the debug flag and just got this error.:

```
Failed to parse AsyncAPI file
[]
```
The empty array seems to be the debug messages.

**Comment by boyney123** [comment]
Thanks can you paste the AsyncAPI file you used please?

**Comment by KatzeMitHut** [comment]
I used the one from the page I linked. 

```
asyncapi: 3.0.0
info:
  title: User Signup API
  version: 1.0.0
  description: The API notifies you whenever a new user signs up in the application.
servers:
  kafkaServer:
    host: test.mykafkacluster.org:8092
    description: Kafka Server
    protocol: kafka
operations:
  onUserSignedUp:
    action: receive
    channel:
      $ref: '#/channels/userSignedUp'
channels:
  userSignedUp:
    description: This channel contains a message per each user who signs up in our application.
    address: user_signedup
    messages:
      userSignedUp:
        $ref: '#/components/messages/userSignedUp'
components:
  messages:
    userSignedUp:
      payload:
        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
        schema:
          type: record
          name: UserSignedUp
          namespace: com.company
          doc: User sign-up information
          fields:
            - name: userId
              type: int
            - name: userEmail
              type: string
              
              
```

**Comment by KatzeMitHut** [comment]
After creating a new event catalog with the latest version, I get this error on 
```
[
  {
    code: 'uncaught-error',
    message: 'Error thrown during AsyncAPI document parsing. Name: Error, message: Unknown schema format, stack: Error: Unknown schema format\n'
```

**Comment by boyney123** [comment]
Should be fixed in `@eventcatalog/generator-asyncapi@0.1.4` let me know if you still have any issues
